xbmc/platform/linux/input/LibInputHandler.cpp

```diff
@@ -13,6 +13,8 @@
 #include "LibInputSettings.h"
 #include "LibInputTouch.h"
 #include "utils/log.h"
+#include "ServiceBroker.h"
+#include "interfaces/AnnouncementManager.h"
```

Comment:
Can you alphabetize the inputs? I think clang-format will do this for you automatically (and if you force-push I think Jenkins will chime in with a diff)